mod_reforged/hooks/skills/actives/miasma_skill.nut

```diff
@@ -0,0 +1,27 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/miasma_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
```

Comment:
I propose we slightly adjust the name of this skill to `Miasma Cloud`.
Just Miasma strikes me as a working title more than anything else. `Miasma` imo is just the effect that lingers on a single tile. While a skill which applies that effect on multiple tiles at once over a distance should have a different name.

Reply:
`Miasma Cloud` sounds equally as much an effect name as `Miasma` itself e.g. the tile has miasma cloud on it or is affected by miasma cloud. If we want to make the skill name more interesting it could be `Summon Miasma` or something along those lines.

But I don't want to change the names of vanilla skills unless there is a very strong reason for it and this sounds very subjective so I'd not touch it.